### PR TITLE
modify get_arrangement API and add exporting csv files

### PR DESCRIPTION
**Pull request by Jeffchiucp**
* add exporting csv files
* modify get_arrangement API

### CODE REVIEW

File: export.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import StringIO
+import csv
+from flask import make_response, jsonify
+
+
+def to_csv(arrangements):
+    si = StringIO.StringIO()
+    cw = csv.writer(si)
+
+    cw.writerow(arrangements.keys())
```
Comment (deonjo) [inline export.py]: sure we can do this for now. But the actual way we want to write out the CSV is very different. But at least there is something in place for now.

File: export.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import StringIO
+import csv
+from flask import make_response, jsonify
+
+
+def to_csv(arrangements):
```
Comment (deonjo) [inline export.py]: move all these methods into the class as static methods

File: export.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    cw.writerow(arrangements.keys())
+    cw.writerow(arrangements.values())
+    output = make_response(si.getvalue())
+    output.headers["Content-Disposition"] = "attachment; filename=arrangements.tsv"
+    output.headers["Content-type"] = "text/csv"
+    return output
+
+
+class Export:
+
```
Comment (deonjo) [inline export.py]: get rid of line

File: export.py (hunk 1 above)
Comment (Jeffchiucp) [reply export.py]: what's you suggestion on writing out the CSV?

**Pull request merged**